### PR TITLE
[vi] Convolutional Neural Networks

### PR DESCRIPTION
**Pull request by rootonchair**
Hi @shervinea, I'm working on translating the Convolutional Neural Networks cheatsheet. Can you label this PR "in progress"?

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you @rootonchair for your help!

File: vi/cs-230-convolutional-neural-networks.md [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+**50. Types of models ― There are 3 main types of object recognition algorithms, for which the nature of what is predicted is different. They are described in the table below:**
+
+&#10230; Các kiểu mô hình ― Có 3 kiểu thuật toán nhận diện vật thể chính, vì thế mà bản chất của thứ được dự đoán sẽ khác nhau. Chúng được miêu tả ở bảng dưới:
+
+<br>
+
+
+**51. [Image classification, Classification w. localization, Detection]**
+
+&#10230; [Phân loại hình ảnh, Phân loại cùng với định vị, Nhận diện]
```
Comment (damminhtien) [inline vi/cs-230-convolutional-neural-networks.md]:
```suggestion
&#10230; [Phân loại hình ảnh, Phân loại cùng với khoanh vùng, Phát hiện]
```

File: vi/cs-230-convolutional-neural-networks.md [hunk 2]
```diff
@@ -0,0 +1,716 @@
...
+**3. [Overview, Architecture structure]**
+
+&#10230; [Tổng quan, Kết cấu kiến trúc]
+
+<br>
+
+
+**4. [Types of layer, Convolution, Pooling, Fully connected]**
+
+&#10230; [Các kiểu tầng (layer), Tích chập, Pooling, Kết nối đầy đủ]
```
Comment (damminhtien) [inline vi/cs-230-convolutional-neural-networks.md]:
```suggestion
&#10230; [Các kiểu tầng (layer), Tích chập, Gộp, Kết nối đầy đủ]
```

File: vi/cs-230-convolutional-neural-networks.md [hunk 3]
```diff
@@ -0,0 +1,716 @@
...
+**7. [Activation functions, Rectified Linear Unit, Softmax]**
+
+&#10230; [Các hàm kích hoạt, Rectified Linear Unit, Softmax]
+
+<br>
+
+
+**8. [Object detection, Types of models, Detection, Intersection over Union, Non-max suppression, YOLO, R-CNN]**
+
+&#10230; [Nhận diện vật thể, Các kiểu mô hình, Detection, Intersection over Union, Non-max suppression, YOLO, R-CNN]
```
Comment (damminhtien) [inline vi/cs-230-convolutional-neural-networks.md]:
```suggestion
&#10230; [Phát hiện vật thể, Các kiểu mô hình, Detection, Intersection over Union, Non-max suppression, YOLO, R-CNN]
```

File: vi/cs-230-convolutional-neural-networks.md [hunk 4]
```diff
@@ -0,0 +1,716 @@
...
+**13. Architecture of a traditional CNN ― Convolutional neural networks, also known as CNNs, are a specific type of neural networks that are generally composed of the following layers:**
+
+&#10230; Kiến trúc truyền thống của một mạng CNN ― Mạng neural tích chập (Convolutional neural networks), còn được biết đến với tên CNNs, là một dạng mạng neural được cấu thành bởi các tầng sau:
+
+<br>
+
+
+**14. The convolution layer and the pooling layer can be fine-tuned with respect to hyperparameters that are described in the next sections.**
+
+&#10230; Tầng tích chập và tầng pooling có thể được hiệu chỉnh theo các tham số cấu hình (hyperparameters) được mô tả ở những phần tiếp theo.
```
Comment (damminhtien) [inline vi/cs-230-convolutional-neural-networks.md]:
```suggestion
&#10230; Tầng tích chập và tầng pooling có thể được hiệu chỉnh theo các siêu tham số (hyperparameters) được mô tả ở những phần tiếp theo.
```

Comment (rootonchair) [reply vi/cs-230-convolutional-neural-networks.md]: Review accepted

File: vi/cs-230-convolutional-neural-networks.md (hunk 2 above)
Comment (rootonchair) [reply vi/cs-230-convolutional-neural-networks.md]: I don't think that is a proper translation for "pooling". Do you have any references?

File: vi/cs-230-convolutional-neural-networks.md (hunk 3 above)
Comment (rootonchair) [reply vi/cs-230-convolutional-neural-networks.md]: Review accepted

File: vi/cs-230-convolutional-neural-networks.md (hunk 1 above)
Comment (rootonchair) [reply vi/cs-230-convolutional-neural-networks.md]: Review accepted

File: vi/cs-230-convolutional-neural-networks.md (hunk 2 above)
Comment (damminhtien) [reply vi/cs-230-convolutional-neural-networks.md]: Can't excerpt the detail but it is quite popular in my university (HUST)

Comment (rootonchair) [reply vi/cs-230-convolutional-neural-networks.md]: Thank you. I think it's better to keep it as "pooling"

**Comment by rootonchair** [comment]
Hi @shervinea, I finished my translation. Can you add "reviewer wanted" label? Thank you in advance.

**Comment by shervinea** [comment]
Awesome @rootonchair, thank you for your work! Changing the label right now.

**Comment by shervinea** [comment]
Thank you @rootonchair and @damminhtien for all your work! Moving forward with the merge!

**Pull request merged**